coroutines-codelab/build.gradle

```diff
@@ -20,14 +20,14 @@ buildscript {
     ext {
 
         // global variables
-        kotlin_version = '1.3.61'
+        kotlin_version = '1.5.31'
```

Comment:
Should we also update the `coroutines_android_version` to 1.5.2? It shouldn't cause any breaking changes

Reply:
Sure. cc @osuleymanova

Reply:
@manuelvicnt 
1.5.2 version caused breaking changes on my side.